### PR TITLE
Update sharethrough bid adapter

### PR DESCRIPTION
**Pull request by chriscpan**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other

## Description of change

Sharetrough bid adapter update for Prebid 1.0

### CODE REVIEW

**Comment by chriscpan** [comment]
Sorry about that, I updated the PR with the .md file. Thanks for the review

File: modules/sharethroughBidAdapter.js [hunk 1]
```diff
@@ -1,128 +1,71 @@
...
+    })
+  },
+  interpretResponse: (resp, req) => {
+    if (!Object.keys(resp).length) return [];
+
+    const creative = resp.creatives[0];
+
+    return [{
+      requestId: req.data.bidId,
+      bidderCode: BIDDER_CODE,
```
Comment (matthewlane) [inline modules/sharethroughBidAdapter.js]: `bidderCode` will be set automatically by `bidderFactory` now, this line can be dropped

File: modules/sharethroughBidAdapter.js [hunk 2]
```diff
@@ -1,128 +1,71 @@
...
+          bidId: bid.bidId,
+          placement_key: bid.params.pkey,
+          hbVersion: '$prebid.version$',
+          strVersion: VERSION,
+          hbSource: 'prebid'
+        }
+      };
+    })
+  },
+  interpretResponse: (resp, req) => {
```
Comment (matthewlane) [inline modules/sharethroughBidAdapter.js]: #1748 changed the first argument of `interpretResponse` to:

```JavaScript
{
  body: responseBody,
  headers: {
    get: function(header) { /* returns a header from the HTTP response */ }
  }
}
```

so adding something like
```JavaScript
resp = resp.body;
```
just below this line, or however you'd prefer to grab the `body`, and updating corresponding tests if needed should get this back to working properly

**Comment by chriscpan** [comment]
Awesome, good to know. Ready for review

**Pull request merged**